### PR TITLE
accounts_password_set_min_life_existing: Avoid system accounts

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- Same principle as PR #9954 but for accounts_password_set_min_life_existing.

### CODE REVIEW

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
The errors in `testing-farm:centos-stream-8-x86_64` and `testing-farm:centos-stream-9-x86_64` seems to be legit. The rule is still failing after the Ansible remediation. Could you check it, please?

**Comment by dodys** [comment]
@marcusburghardt I'm not really sure why it is failing for fedora and CentOS, do you have any idea?

**Comment by marcusburghardt** [comment]
> @marcusburghardt I'm not really sure why it is failing for fedora and CentOS, do you have any idea?

No yet but I can take a look on this.

**Comment by freddieRv** [comment]
Hi! Doing some testing on our side with these commits backported we found an issue were the `correct_stig_noninteractive.pass.sh` test results in a false negative when there are no interactive users on the system (other than root).

This issue is also present on the `accounts_password_set_max_life_existing` rule with the changes from #9954 

Changing the `check_existence` attribute of the OVAL tests to `any_exist` fixes this. However with this change the `incorrect_min_pass_age.fail.sh` test reports that "_Rule evaluation resulted in pass, instead of expected fail during initial stage_". An interactive user could be added on this test to make sure there is at least one ill-configured user to fail the test.

**Comment by dodys** [comment]
> Hi! Doing some testing on our side with these commits backported we found an issue were the `correct_stig_noninteractive.pass.sh` test results in a false negative when there are no interactive users on the system (other than root).
> 
> This issue is also present on the `accounts_password_set_max_life_existing` rule with the changes from #9954
> 
> Changing the `check_existence` attribute of the OVAL tests to `any_exist` fixes this. However with this change the `incorrect_min_pass_age.fail.sh` test reports that "_Rule evaluation resulted in pass, instead of expected fail during initial stage_". An interactive user could be added on this test to make sure there is at least one ill-configured user to fail the test.

Thanks Freddie for testing this PR. Indeed your suggestions make sense, so I add a new commit changing the existence check to `any_exist` and simplified to `incorrect_min_pass_age.fail.sh` test.
Let me know in case of issues.
I will be doing the same for max life.

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
Please, take a look in the comments from #9954

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh [hunk 1]
```diff
@@ -1,13 +1,7 @@
 #!/bin/bash
 
-# packages = passwd
-
 BAD_PAS_AGE=-1
 
-# Configure the OS to enforce a password age < 1 of each accout  
+useradd testuser_123
```
Comment (marcusburghardt) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/tests/incorrect_min_pass_age.fail.sh:5]:
```suggestion
USERNAME="testuser_123"
useradd $USERNAME
echo "cac_test_pass" | passwd --stdin $USERNAME
```

**Review by marcusburghardt — APPROVED** [review]
Thanks for the improvements in this rule @dodys .

**Pull request merged**